crates/solver/src/settlement_submission/submitter.rs

```diff
@@ -325,6 +338,16 @@ impl<'a> Submitter<'a> {
                 }
             };
 
+            // before submitting, check if the currently executing strategy is temporarily disabled
+
+            if let SubmissionLoopStatus::Disabled(reason) =
+                self.submit_api.submission_status(&settlement)
+            {
+                tracing::debug!("strategy temporarily disabled, reason: {:?}", reason);
+                tokio::time::sleep(params.retry_interval).await;
+                continue;
```

Comment:
Is the reasoning here that a strategy may be temporarily disabled and then re-enabled in the future? Is this something that we should account for now or can we just `break` when a strategy becomes disabled?

Reply:
Exactly, enable/reenable is the idea behind the `continue`. At this moment I don't see any downside of it, and also I don't see any benefit of ending the submission loop for currently executing strategy.